### ISSUE TITLE
Add lang attribute to data service

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1024]
Data service html is missing `lang` attribute

Open SDG platform html element:
![image](https://user-images.githubusercontent.com/43404358/102094318-18bc5680-3e1a-11eb-8173-6d056ae2360e.png)

UK data service html element:
![image](https://user-images.githubusercontent.com/43404358/102094381-2b369000-3e1a-11eb-9ecc-cc9ad7d6f10f.png)

This is an A accessibility issue
